docs/hdfs_connector.rst

```diff
@@ -95,13 +150,17 @@ You should see the following output::
 
 .. note:: If you want to run the Quickstart with Hive integration, before starting the connector,
```

Comment:
Again, I'm suggesting addressing this one separately so it won't be hidden under a commit related to the CLI.